src/config/routes.config.tsx

```diff
@@ -0,0 +1,22 @@
+import { RouteObject } from "react-router-dom";
+import { LandingPage, TripDetailPage, TripsListingPage } from "../pages";
+import { ErrorPage } from "../pages/ErrorPage/ErrorPage";
+
+export const routesConfig: RouteObject[] = [
+  {
+    path: "/",
+    element: <LandingPage />,
+  },
+  {
+    path: "/trips",
```

Comment:
Better to move strings routes ("/trips") to constants and import it.

Reply:
Done